Tools/autotest/ardusub.py

```diff
@@ -879,7 +944,8 @@ def tests(self):
             self.MAV_CMD_CONDITION_YAW,
             self.TerrainMission,
             self.SetGlobalOrigin,
-            self.backup_home,
+            self.backup_origin,
+            self.fuse_mag,
```

Comment:
Just as a future cleanup, these don't match our naming convention for tests, which you can see in the list above.

Concept is that you can visually distinguish helper functions from tests (kind of entry points, really) that way.

Reply:
Ack, I was confused about naming, thanks for clearing this up